packages/storybook-html/src/components/core/date.story.mdx

```diff
@@ -0,0 +1,285 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+
+<Meta title="Core/Standards/Date formats" />
+
+# Date Formats
+
+## Overview
+
+Display data such as date & time in a consistent and user-friendly way. Present the information concisely and avoid unnecessary redundancy.
+
+These date formats are designed for
+
+* The current date and time
+* A date and time setting, such as an alarm or event reminder
+* A date and time when an object was created, sent, edited, or otherwise acted upon
+
+## Basic displays
+
+<table>
+    <caption>Basic date displays</caption>
+    <thead>
+        <tr>
+          <th>Example</th>
+          <th>Format</th>
+          <th>Usage</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+          <td>
+          10:00 AM <br />
+          2:30 PM
+          </td>
+          <td><code>hh:mm AM/PM</code></td>
+          <td>For time, use uppercase AM or PM without periods, separated from time with a space. <br />
```

Comment:
need to probably confirm this whether upper or lower case

Reply:
I will add a ticket to review this with the content team as we have conflicting advice.